app/build.gradle

```diff
@@ -42,4 +48,7 @@ dependencies {
     implementation "org.jetbrains.anko:anko-recyclerview-v7:$anko_version"
     implementation 'de.hdodenhof:circleimageview:3.0.1'
     implementation 'com.github.bumptech.glide:glide:4.10.0'
+
+    implementation 'com.squareup.picasso:picasso:2.71828'
```

Comment:
This library is not use, you can remove it

Reply:
resolved, thanks